vector/src/main/java/im/vector/KeyRequestHandler.java

```diff
@@ -358,7 +373,7 @@ public void onSuccess(Void info) {
         mAlertDialog.setOnCancelListener(new DialogInterface.OnCancelListener() {
             @Override
             public void onCancel(DialogInterface dialogInterface) {
-                onDisplayKeyShareDialogClose(false);
+                onDisplayKeyShareDialogClose(false, true);
```

Comment:
2nd param should be false in this case, no? User does not want explicitly ignore the request.
Nevermind I think this code is useless cause the dialog is not cancellable

Reply:
Well, I did not know what setOnCancelListener does. Sorry :|
I let you update it if needed.